anypoint-mq/4.x/modules/ROOT/pages/anypoint-mq-failover.adoc

```diff
@@ -21,6 +21,10 @@ To use the failover feature, do the following:
 * In Anypoint MQ:
 .. Create or modify a queue and xref:mq::mq-failover.adoc#enable-failover[enable failover on the queue].
 
+[NOTE]
+With failover enabled, the app can't consume directly from the fallback queue.
```

Comment:
Should we provide some reasoning for why we do this? When failover system property is enabled, MQ Connector will handle processing messages from both the primary and fallback queues and apps will not be allowed to consume from the fallback queue directly to avoid interrupting the processing that the Connector is doing.

Reply:
Yes, but not in a note, and maybe not here. I think it belongs in the other section...